test/bats/oathkeeper.bats

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bats
+
+load "helpers/setup-and-teardown"
+
+setup_file() {
+  start_server
+}
+
+teardown_file() {
+  stop_server
+}
+
+decisionsApi=$(cat galoy.yaml| grep decisionsApi | awk '{print $2}')
+OATHKEEPER_ENDPOINT=${decisionsApi:-"http://127.0.0.1:4456/decisions/"}
+
+check_is_uuid() {
+  uuid_string=$1
+
+  uuid_regex='^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
+  echo $uuid_string | grep -Eq "$uuid_regex"
+}
+
+decode_jwt() {
+  local jwt="$1"
+  local part="$2"
+
+  # Split JWT into parts
+  local header="$(echo "$jwt" | cut -d "." -f 1)"
+  local payload="$(echo "$jwt" | cut -d "." -f 2)"
+  local signature="$(echo "$jwt" | cut -d "." -f 3)"
+
+  # Helper to decode Base64Url
+  decode_base64url() {
+    local data="$1"
+    local len=${#data}
+
+    # Add padding
+    local mod=$((len % 4))
+    if [[ $mod -eq 2 ]]; then data="${data}=="; fi
+    if [[ $mod -eq 3 ]]; then data="${data}="; fi
+
+    # Translate base64url to base64
+    local url_decoded="$(echo "$data" | tr '_-' '/+')"
+
+    echo "$url_decoded" | base64 --decode
+  }
+
+  case "$part" in
+    header)
+      decode_base64url "$header"
+      ;;
+    payload)
+      decode_base64url "$payload"
+      ;;
+    signature)
+      # Not decoding signature, just showing it
+      echo "$signature"
+      ;;
+    *)
+      echo "Invalid part. Choose header, payload, or signature."
+      ;;
+  esac
+}
+
+exec_oathkeeper() {
+  local token_name=$1
+
+  if [[ ${token_name} == "anon" ]]; then
+    AUTH_HEADER=""
+  else
+    AUTH_HEADER="Authorization: Bearer $(read_value ${token_name})"
+  fi
+
+  if [[ "${BATS_TEST_DIRNAME}" != "" ]]; then
+    run_cmd="run"
+  else
+    run_cmd=""
+  fi
+
+  ${run_cmd} curl -s -I \
+    -X POST \
+    ${AUTH_HEADER:+ -H "$AUTH_HEADER"} \
+    -H "Content-Type: application/json" \
+    "${OATHKEEPER_ENDPOINT}graphql"
```

Comment:
This looks weird. should be `{OATHKEEPER_ENDPOINT}/graphql`

Reply:
Yea it's because of how we've already configured it in the trailing slash in `galoy.yaml` and are using it in other places. I wasn't sure if changing this would affect things in other places or custom configs so I left it as is